src/drivers/ist8310/ist8310.cpp

```diff
@@ -1249,164 +1215,6 @@ IST8310::set_selftest(unsigned enable)
 	return !(str == str_reg_ret);
 }
 
-bool
```

Comment:
Neither @julianoes nor I do know why the code is there in the first place. Since it's not used, I think it's fine to remove it.